src/group_theory/coset.lean

```diff
@@ -303,6 +303,21 @@ quotient_group.eq
 lemma out_eq' (a : quotient s) : mk a.out' = a :=
 quotient.out_eq' a
 
+variables (s)
+
+/- It can be useful to write `obtain ⟨g, h⟩ := mk_out'_eq_mul ...`, and then `rw [h]` or
+  `simp_rw [h]` or `simp only [h]`. In order for `simp_rw` and `simp only` to work, this lemma is
+  stated in terms of an arbitrary `h : s`, rathern that the specific `h = g⁻¹ * (mk g).out'`. -/
```

Comment:
The `g` and first (but not second) `h` you're using in this docstring are different `g` and `h` to the one in the lemma statement, which is unnecessarily confusing.

Reply:
Ah, good point.

Reply:
Hopefully it's clearer now.